lib/zclient.h

```diff
@@ -38,6 +38,15 @@
 
 #include "mlag.h"
 
+/* Zebra types. Used in Zserv message header. */
+typedef uint16_t zebra_size_t;
```

Comment:
Couldn't we remove this typedef instead? It's being used quite inconsistently and doesn't add any value in my opinion. Other than that, this looks good to me.

Reply:
(we can still do that later, in the meantime i'll go merge this)